### DISCUSSION TITLE
Acesso a Dados com Dapper.

### DISCUSSION THREAD

**Discussion by samueldevadn** [balta-io/forum#387] [Tira Dúvidas] [answered]
Fala galera, tudo bem?
Estou no modelo de acesso a dados com o dapper e quando executo um "dotnet run" no terminal, ele me retorna um erro estranho
Vcs podem me ajudar?

![image](https://github.com/balta-io/forum/assets/128742118/df8ce64e-ca5d-4941-87e6-1bb0fabd7403)
.

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @samueldevadn, como vai?

O erro ficou oculto na imagem que você mandou!

**Reply by samueldevadn** [reply]
Não é bem um erro, quando eu executo o código, o terminal mostra essa mensagem mas não exibe os dados.

**Reply by samueldevadn** [reply]
E estava conectado tudo certinho, o container la no docker, a string de conexão. Tudo certinho

**Reply by andrebaltieri** [reply]
Então, mas preciso ver os detalhes, ficou oculto no print que você enviou.
